src/plugins/data/common/field_formats/converters/url.ts

```diff
@@ -64,8 +64,8 @@ export class UrlFormat extends FieldFormat {
   ];
   static urlTypes = URL_TYPES;
 
-  constructor(params: Record<string, any>) {
-    super(params);
+  constructor(params: Record<string, any>, getConfig: Function) {
```

Comment:
Does this converter need the config?
Or is this a left over from the previous version?